buildSrc/src/main/java/org/elasticsearch/gradle/Jdk.java

```diff
@@ -78,6 +80,21 @@ public void setPlatform(String platform) {
         this.platform.set(platform);
     }
 
+    private static final Set<String> ALLOWED_VENDORS = Set.of("adoptopenjdk", "openjdk");
```

Comment:
nit: Can we move these up with the other instance variables and static members?

Reply:
I pushed a0ae195a3d135f45f31307343f5b6d298c1686c4.